nestjs/src/config/config.service.ts

```diff
@@ -76,5 +81,12 @@ export class ConfigService {
       hirers: conf.get('RSSHCOOL_USERS_HIRERS')?.split(',') ?? [],
       admins: conf.get('RSSHCOOL_USERS_ADMINS')?.split(',') ?? [],
     };
+
+    const DEV_HOST = conf.get('RSSCHOOL_CROSS_CHECK_SCHEDULING_DEV_HOST') ?? '';
```

Comment:
this looks confusing. Why do we need separate config values for `prod` and `dev`? and what's about e.g. feature branches ?

Reply:
if you just need host, please see `RSSHCOOL_HOST`  and you can port it to `nestjs`. I would avoid creating a new config values

Reply:
replaced wth `RSSCHOOL_HOST`